Formula/c/checkov.rb

```diff
@@ -7,6 +7,12 @@ class Checkov < Formula
   sha256 "dca00fe8f0e04fd4fde88de5143bbe01aae7966351cd0f70f69456313bd43428"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://pypi.org/rss/project/checkov/releases.xml"
+    regex(%r{<title>v?(\d+(?:\.\d+)+)</title>}i)
```

Comment:
Or git as I mentioned in #166497
```suggestion
    url "https://github.com/bridgecrewio/checkov.git"
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

`git` is probably safer in avoiding risk of multiple pre-releases (alpha/beta/rc) that can fill up RSS feed, though `checkov` never did pre-releases before (even going from v2 to v3).

PyPI RSS is currently faster as less content. Running `git ls-remote --tags` is 199KB (and will continue to increase) while RSS xml is 11KB (should remain relatively same as there is a limit). 

Another option is GitHub API, but we probably need to be more careful due to autobump being at risk of hitting rate limits.